src/core/PWScore.cpp

```diff
@@ -2528,6 +2545,18 @@ bool PWScore::Validate(const size_t iMAXCHARS, CReport *pRpt, st_ValidateResults
         pRpt->WriteLine(cs_Error);
       } );
     }
+
+    // Passkey-related issues
+    if (st_vr.num_incomplete_passkeys > 0) {
```

Comment:
Could we define a separate method to write passkey issues to `pRpt`?

Reply:
Yeah, will look at that tomorrow, separating sounds like a good idea.

Reply:
I've moved the report formatting to Report.cpp.

Let me know if you like it.